packages/ui/src/2_molecules/VerticalTabs/components/VerticalTabItem.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
+
+import classNames from 'classnames';
+
+import { VerticalTabsItemButtonProps } from '../VerticalTabs.types';
+import styles from './VerticalTabItem.module.css';
+
+export const VerticalTabItem: FC<VerticalTabsItemButtonProps> = ({
+  disabled,
+  label,
+  infoText,
+  active,
+  dataActionId,
```

Comment:
i think makes sense to change it now dataActionId -> dataLayoutId and ping Alita or Luna to update test cases